EAD/Session8/Assignments/studentSpringMVC/src/main/java/com/metacube/student/dao/StudentDAO.java

```diff
@@ -0,0 +1,60 @@
+package com.metacube.student.dao;
+import java.util.ArrayList;
+import java.util.List;
+import org.springframework.stereotype.Repository;
+
+import com.metacube.student.model.StudentModel;
+
+/**
+ * 
+ * @author Nilesh Patel
+ *
+ */
+@Repository
+public class StudentDAO 
+{
+
+	static List<StudentModel> students = new ArrayList<StudentModel>();
+	static int counter = 1;
+
+	public int addStudent(StudentModel student) 
```

Comment:
Where is ```ValidationMessages.properties``` and ```application.yml```

Reply:
I forgot to upload both files. Now I uploaded.